src/components/date-picker-month-header/date-picker-month-header.tsx

```diff
@@ -259,27 +264,50 @@ export class DatePickerMonthHeader {
     this.calciteDatePickerSelect.emit(date);
   };
 
-  /**
-   * Parse localized year string from input,
-   * set to active if in range
-   */
-  private setYear(localizedYear: string, increment = 0) {
-    const { min, max, activeDate, localeData, yearInput } = this;
+  private getInRangeDate({
+    localizedYear,
+    increment = 0
+  }: {
+    localizedYear: string;
+    increment?: number;
```

Comment:
Nitpick: is there another name we can use for this var? `increment: -1` threw me off. Same for `setYear`.

Reply:
Yeah, not sure on this one. maybe offset? I'll change to that for now.